adjustText/adjustText.py

```diff
@@ -19,20 +19,18 @@ def get_midpoint(bbox):
     cy = (bbox.y0+bbox.y1)/2
     return cx, cy
 
-def find_renderer(fig):
-    if hasattr(fig.canvas, "get_renderer"):
-        renderer = fig.canvas.get_renderer()
-    else:
-        renderer = fig.canvas.renderer
-
-    return renderer
-
 def get_points_inside_bbox(x, y, bbox):
     x1, y1, x2, y2 = bbox.xmin, bbox.ymin, bbox.xmax, bbox.ymax
     x_in = np.logical_and(x>x1, x<x2)
     y_in = np.logical_and(y>y1, y<y2)
     return np.where(x_in & y_in)[0]
 
+def get_renderer(fig):
+    try: 
+        return fig.canvas.get_renderer()
+    except:
```

Comment:
`except AttributeError:` is preferable, since we don't know what other kinds of errors there might be raised.